packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionSmoothers_def.hpp

```diff
@@ -413,16 +413,19 @@ void chebyshevIterate ( RCP<Teuchos::ParameterList> params,
   const int maxIter      = params->get<int>   ("smoother: sweeps");
   const Scalar eigRatio  = params->get<double>("smoother: eigRatio");
   const Scalar lambdaMax = params->get<Scalar>("chebyshev: lambda max");
-  const Scalar lambdaMin = lambdaMax / eigRatio;
+  const Scalar boostFactor = 1.1; // TODO: add as a param
```

Comment:
@pohm01 You could read this parameter from the command line in the driver and then pass it thought the parameter list as entry `chebyshev: boost factor`.

Reply:
Added the command line argument `--smootherChebyBoostFactor` to control this parameter from the command line.